ansible_collections/arista/avd/molecule/evpn_underlay_ebgp_overlay_ebgp/documentation/fabric/DC1_FABRIC-topology.csv

```diff
@@ -3,11 +3,17 @@ l3leaf,DC1-BL1A,Ethernet1,spine,DC1-SPINE1,Ethernet6
 l3leaf,DC1-BL1A,Ethernet2,spine,DC1-SPINE2,Ethernet6
 l3leaf,DC1-BL1A,Ethernet3,spine,DC1-SPINE3,Ethernet6
 l3leaf,DC1-BL1A,Ethernet4,spine,DC1-SPINE4,Ethernet6
+l3leaf,DC1-BL1A,Ethernet7,,,
```

Comment:
These CSVs reflect the topology. We should at least add some static "peer_type" key on the generated structured config. Ex. "l3_interface"